flink-python/pyflink/table/table.py

```diff
@@ -1077,6 +1078,89 @@ def explain(self, *extra_details: ExplainDetail) -> str:
         j_extra_details = to_j_explain_detail_arr(extra_details)
         return self._j_table.explain(TEXT, j_extra_details)
 
+    def insert_into(
+        self, table_path_or_descriptor: Union[str, TableDescriptor], overwrite: bool = False
+    ) -> TablePipeline:
+        """
+        When ``target_path_or_descriptor`` is a table path:
+
+            Declares that the pipeline defined by the given :class:`Table` (backed by a
```

Comment:
This doc has not been copied correctly. It doesn't make much sense.

Reply:
Is it not the docs for `Table.insertInto`? https://nightlies.apache.org/flink/flink-docs-release-2.0/api/java/org/apache/flink/table/api/Table.html#insertInto(java.lang.String)

Reply:
The `(backed by a ...)` belongs after the `written to a table`.